sky/services/media/media.mojom

```diff
@@ -14,7 +14,23 @@ interface MediaPlayer {
   SeekTo(uint32 msec);
 };
 
+[ServiceName="media::SoundPool"]
+interface SoundPool {
+  Load(handle<data_pipe_consumer> data_source) => (bool success, int32 sound_id);
+  Play(int32 sound_id, float left_volume, float right_volume, bool loop, float rate) =>
+      (bool success, int32 stream_id);
+  Stop(int32 stream_id);
+  Pause(int32 stream_id);
+  Resume(int32 stream_id);
+  SetRate(int32 stream_id, float rate);
+  SetVolume(int32 stream_id, float left_volume, float right_volume);
```

Comment:
This is what's supported in the underlying SoundPool API in Android.

We could conceivably support more channels by using lower-level APIs like AudioTrack, but that's a more complex project.